app/assets/javascripts/discourse/templates/components/time-input.hbs

```diff
@@ -0,0 +1,35 @@
+<div class="fields">
+  {{#if nativePicker}}
+    {{!-- {{input class="field time" type="time" value=_time}} --}}
+  {{else}}
+    {{input
+      class="field hours"
+      type="number"
+      title="Hours"
+      minlength=2
+      maxlength=2
+      max="23"
+      min="0"
+      placeholder="00"
+      value=_hours
+      input=(action "onInput" (hash prop="hours"))
+      focus-in=(action "onFocusIn")
+    }}
+
+    <div class="separator">:</div>
```

Comment:
I wonder, is time universally written as `HH:mm` or are there locales where it isn't?

Reply:
I would want to start with this, and improve along the way as I implement it in the various UIs/cases we have using it